src/constants/java/net/fabricmc/yarn/constants/WorldEvents.java

```diff
@@ -0,0 +1,558 @@
+package net.fabricmc.yarn.constants;
+
+/**
+ * Constants of World Event IDs.<br>
+ * World Events are used to trigger things on the client from the server side.
+ * Most commonly, playing sound events or spawning particles.<br>
+ * Some events have an extra data integer sent alongside them.<br>
+ * Some events are global, meaning they will be sent to every player regardless of their position.<br>
+ * Events are sent from the server to the client using {@link net.minecraft.network.packet.s2c.play.WorldEventS2CPacket WorldEventS2CPacket},
```

Comment:
`<p>` would be better than `<br>` here. (Also possibly when separating the 2nd description line of the individual constants from the "called by" sections)